plugins/org/src/components/Cards/Group/MembersList/MembersListCard.tsx

```diff
@@ -245,13 +248,22 @@ export const MembersListCard = (props: {
     />
   );
 
+  const filteredMembers = members.filter(member =>
+    member.spec.profile?.displayName
```

Comment:
Not everyone has `displayName` set.

The IDEAL solution would be to use the `presentationApi` to get what the actual correct display name is for each entity, and then filter by that. That ensures that the filter matches what you actually are presented with on the screen.

But to keep it simple, I'd say match on the term existing in any of

* `metadata.name`
* `metadata.title`
* `spec.profile?.displayName`
* `spec.profile?.email`